libcloud/ecp.cfg.orig

```diff
@@ -56,6 +56,17 @@ libcloud:
             - zypper in -y lsb-release make gcc gcc-c++ chrony || true
             - systemctl enable chronyd.service || true
             - systemctl start chronyd.service || true
+            - sed -i 's/NETCONFIG_DNS_STATIC_SEARCHLIST=""/NETCONFIG_DNS_STATIC_SEARCHLIST="{% raw %}{{ lab_domain }}{% endraw %}"/' /etc/sysconfig/network/config
+            - sed -i 's/NETCONFIG_DNS_STATIC_SERVERS=""/NETCONFIG_DNS_STATIC_SERVERS="{% raw %}{{ nameserver_addr }}{% endraw %}"/' /etc/sysconfig/network/config
+        "sle-15.2":
+          runcmd:
+            - *resolv_conf
+            - zypper in -y git wget rsyslog || true
+            - zypper in -y lsb-release make gcc gcc-c++ chrony || true
+            - systemctl enable chronyd.service || true
+            - systemctl start chronyd.service || true
+            - sed -i 's/NETCONFIG_DNS_STATIC_SEARCHLIST=""/NETCONFIG_DNS_STATIC_SEARCHLIST="{{ lab_domain }}"/' /etc/sysconfig/network/config
+            - sed -i 's/NETCONFIG_DNS_STATIC_SERVERS=""/NETCONFIG_DNS_STATIC_SERVERS="{{ nameserver_addr }}"/' /etc/sysconfig/network/config
```

Comment:
Why not to add to `resolv_conf` definition above, it would be better to have as a macro to avoid code duplication